### ISSUE TITLE
verdichten macht Probleme

### ISSUE THREAD

**Issue by Nentorro** [Courseplay/courseplay#2115]
Hi, 
leider macht CP Probleme beim verdichten. Ich fahre den kurs ein wie gewohnt. so klappt er auch. aber sobald ich CP fenster etwas einstellen möchte. stürtzt CP ab. Ich kann dann nicht mehr das Spiel speichern oder zu anderen Fahrzeugen wechseln. ESC klappt dann auch nicht mehr.

Hat noch wer dieses Problem?  Ich habe die aktuelle CP Version drauf

**Comment by pops64** [comment]
https://github.com/Courseplay/courseplay/wiki need a full log. If you can please keep it in English thanks

**Comment by Nentorro** [comment]
If the tractor goes off the normal course, everything is still in order. If I leave the driver dismissed, nothing works. ESC no longer works and the game no longer responds. I can only close it over alt + f4 or the Taskmanager.

**Comment by Nentorro** [comment]
here is my Log 👍 [https://gist.github.com/Nentorro/7a2453706c9094bf9037270b4692c82f](url)

**Comment by pops64** [comment]
Try turning off all vehicle mods. And turn them back on one by one. My guess is you have a bad vehicle mod.

**Comment by Nentorro** [comment]
Yes it was a mod conflict. I have disabled the Coolamon Chaser Bins 30T and 36T, combine addon, AgriCAD pack. Since then it is again
